crates/bevy_render/src/camera/camera.rs

```diff
@@ -915,6 +922,10 @@ pub fn extract_cameras(
             if let Some(perspective) = projection {
                 commands.insert(perspective.clone());
             }
+
+            if gpu_culling && *gpu_preprocessing_support == GpuPreprocessingSupport::Culling {
```

Comment:
```suggestion
            if gpu_culling && *gpu_preprocessing_support == GpuPreprocessingSupport::Culling {
	            warn_once!("GpuCulling not supported on this platform");
```

Could be useful to have a warning if the feature is requested but not supported instead of silently ignoring it.

Reply:
Added it.